docs/users-guide/expressions.md

```diff
@@ -0,0 +1,102 @@
+| Standard deviation | `StandardDeviation(column)`                    | Calculates the standard deviation of the column.                                                                                                                   | `StandardDeviation( [Population] )`                                  |
+| Substring          | `substring(text, position, length)`            | Returns a portion of the supplied text.                                                                                                                            | `substring( [Title], 0, 10 )`                                        |
+| Sum                | `sum(column)`                                  | Adds up all the values of the column.                                                                                                                              | `sum( [Subtotal]` )                                                  |
+| Sum if             | `sumif(column, condition)`                     | Sums up the specified column only for rows where the condition is true.                                                                                            | `sumif( [Subtotal], [Order Status] = "Valid" )`                      |
+| Square root        | `sqrt(number)`                                 | Returns the square root.                                                                                                                                           | `sqrt([Hypotenuse])`                                                 |
+| Trim               | `trim(string)`                                 | Removes leading and trailing whitespace from a string of text.                                                                                                     | `trim( [Comment] )`                                                  |
+| Upper              | `upper(text)`                                  | Returns the text in all upper case.                                                                                                                                | `upper( [Status] )`                                                  |
+| Variance           | `variance(column)`                             | Returns the numeric variance for a given column.                                                                                                                   | `variance([Temperature])`                                            |
+
+### Database limitations
+
+Certain database types don't support some of the above functions:
+
+MySQL
+
+- median
+- percentile
+
+SQLite
+
+- log
+- median
+- percentile
+- power
+- standardDeviation
+- sqrt
+- variance
+
+Additionally, **Vertica**, **BigQuery**, and **Presto** only provide _approximate_ results for median and percentile.
```

Comment:
Do we have specific references we can link out to here? For example, I see that BigQuery has both [`PERCENTILE_CONT`](https://cloud.google.com/bigquery/docs/reference/standard-sql/functions-and-operators#percentile_cont) and [`APPROX_QUANTILES`](https://cloud.google.com/bigquery/docs/reference/standard-sql/functions-and-operators#approx_quantiles). So does this mean that we're using the latter? The former doesn't appear to be approximate.

Reply:
🤷‍♂ paging @sbelak

Reply:
OK, I see we're using `APPROX_QUANTILES` - Will comment in the other ticket to see what the reasoning is there and we can leave this as is for now.